layouts/partials/deprecation-warning.html

```diff
@@ -3,8 +3,9 @@
   <main>
     <div class="content deprecation-warning">
       <h3>
-        Documentation for Kubernetes {{ .Param "version" }} is no longer actively maintained. The version you are currently viewing is a static snapshot.
-        For up-to-date documentation, see the <a href="{{ .Site.Params.currentUrl }}">latest</a> version.
+	 Kubernetes {{ .Param "version" }}
+	 {{ T "deprecation_warning" }}
+	 <a href="{{ .Site.Params.currentUrl }}">{{ T "main_latest" }}</a>。
       </h3>
```

Comment:
@tengqm, do you need `version` after `latest`? I was unable to preview the deprecation warning, as those versions are not re-generated?

Reply:
Sure. The original deprecation string ended as `... latest version.` instead of just `latest`.